test-packages/integration-tests/package.json

```diff
@@ -20,11 +20,14 @@
     "@sap-cloud-sdk/util": "^1.18.1"
   },
   "devDependencies": {
+    "axios": "0.19.2",
     "@types/jest": "^25.1.4",
+    "jsonwebtoken": "^8.5.1",
+    "bignumber.js": "8.1.1",
+    "nock": "^12.0.2",
```

Comment:
Is this really necessary? As far as I understand if this is available on root level we don't need to add it to the packages, except for executables.

Reply:
I tried to remove it and got compilation errors.